lib/airbrake/passenger/integration/class_methods.rb

```diff
@@ -0,0 +1,28 @@
+require 'airbrake/passenger/integration'
+module Airbrake
+  module Passenger
+    module Integration
+      # Module to be extended into ::PhusionPassenger::LoaderSharedHelpers
+      module ClassMethods
+        def self.extended(klass)
+          if klass.respond_to?(:about_to_abort)
+            klass.send :alias_method, :about_to_abort_without_notifier, :about_to_abort
+          end
+        end
+
+        def about_to_abort(*args)
+          exception = args.last if args.last.is_a?(Exception)
+          if exception
+            params = {}
+            params[:component] = "PhusionPassenger " \
+              "v#{::PhusionPassenger::VERSION_STRING}"
```

Comment:
@kyrylo probably this should be appended to `context[:version]`, e.g. `1.2.3 Rails/4.1.6 PhusionPassenger/1.1.1`.